tests/vkray/raygen.slang.glsl

```diff
@@ -1,58 +1,30 @@
-//TEST_IGNORE_FILE:
 #version 460
-
+#extension GL_NV_ray_tracing : require
 layout(row_major) uniform;
+layout(row_major) buffer;
+
+#line 5 "tests/vkray/raygen.slang"
+layout(binding = 0)
+uniform texture2D samplerPosition_0;
 
-#extension GL_NV_ray_tracing : require
 
-#define TRACING_EPSILON 1e-6
-
-#define tmp_ubo             _S1
-#define tmp_saturate        _S2
-#define tmp_launchID_x      _S3
-#define tmp_add_x           _S4
-#define tmp_launchSize_x    _S5
-#define tmp_div_x           _S6
-#define tmp_launchID_y      _S7
-#define tmp_add_y           _S8
-#define tmp_launchSize_y    _S9
-#define tmp_div_y           _S10
-#define tmp_tex_pos         _S11
-#define tmp_tex_nrm         _S12
-#define tmp_light_invDist   _S13
-#define tmp_trace_A         _S14
-#define tmp_trace_B         _S15
-#define tmp_trace_C         _S16
-#define tmp_trace_D         _S17
-#define tmp_trace_E         _S18
-#define tmp_trace_ray       _S19
-#define tmp_trace_payload   _S20
-#define tmp_cmp             _S21
-#define tmp_color           _S22
-#define tmp_dot             _S23
-#define tmp_sat             _S24
-#define tmp_trace2_A        _S25
-#define tmp_trace2_B        _S26
-#define tmp_trace2_C        _S27
-#define tmp_trace2_D        _S28
-#define tmp_trace2_E        _S29
-#define tmp_trace2_ray      _S30
-#define tmp_trace2_payload  _S31
-#define tmp_storeIdx        _S32
```

Comment:
I'm not happy to see all of this work go out the window. I'd set up this `.slang.glsl` file with the hope that it would be easy enough to change the Slang-generated temporary names that match these `#define`s in order to get the code back to working after a change to the GLSL emit logic.

For a larger test, it is always a bit worrying to just check in a new GLSL baseline to use for comparison, because we might not always take the time to carefully validate that GLSL against what it should be. Keeping the GLSL more or less the same, and only allowing changes to temporary variable names helps ensure that we don't introduce larger changes without realizing it.

If actually getting this back into the format it had before is out of the question, then I can live with checking in this new baseline, so long as it has been carefully compared against what we expect.

Reply:
Fixed.